### PR TITLE
Talk and Blog post about Template-Oriented-Programming

### PR DESCRIPTION
**Pull request by overfullstack**
Talk and Blog post about Template-Oriented-Programming using Λrrow Typeclasses.

### CODE REVIEW

File: content/_posts/2020-04-06-template-oriented-programming-talk.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Template-Oriented-Programming to Ship Faster, Part-1
+header-image: https://www.dropbox.com/s/249l72ehauel1np/guanlet-2.gif
```
Comment (rachelcarmena) [inline content/_posts/2020-04-06-template-oriented-programming-talk.md:3]:
```suggestion
header-image: https://img.youtube.com/vi/_QBlKtUY6ac/maxresdefault.jpg
```

Following `https://img.youtube.com/vi/<ID>/maxresdefault.jpg` pattern so we don't depend on Dropbox accounts :+1:

File: content/_posts/2020-04-06-template-oriented-programming.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Template-Oriented-Programming to Ship Faster
+header-image: https://www.dropbox.com/s/249l72ehauel1np/guanlet-2.gif
```
Comment (rachelcarmena) [inline content/_posts/2020-04-06-template-oriented-programming.md:3]: The previous suggested image can be used here as well.

File: content/_posts/2020-04-06-template-oriented-programming-talk.md (hunk 1 above)
Comment (overfullstack) [reply content/_posts/2020-04-06-template-oriented-programming-talk.md]: Thanks, Done!

File: content/_posts/2020-04-06-template-oriented-programming.md (hunk 2 above)
Comment (overfullstack) [reply content/_posts/2020-04-06-template-oriented-programming.md]: Thanks, Done!

**Pull request merged**